src/main/webapp/app/shared/date-time-picker/date-time-picker.component.html

```diff
@@ -43,3 +43,6 @@
 @if (dateInput.invalid || (requiredField && !dateInput.value)) {
     <div class="invalid-feedback">{{ 'entity.dateMissingOrNotValid' | artemisTranslate: { labelName: labelName } }}</div>
 }
+@if (warning) {
+    <div class="invalid-feedback warning" jhiTranslate="entity.warningError" ngbTooltip="{{ 'entity.warningToolTip' | artemisTranslate }}"></div>
```

Comment:
You can use `text-warning`, so you don't need to create an extra `warning` class. However, displaying yellow text on a white background is hardly readable. We should consider other ways to display the warning message, such as displaying a yellow warning triangle and keeping the text in black.